docs/example-with-express.md

```diff
@@ -201,30 +201,34 @@ import {User} from "./User";
 // create typeorm connection
 createConnection().then(connection => {
     const userRepository = connection.getRepository(User);
-    
+
     // create and setup express app
     const app = express();
     app.use(bodyParser.json());
-    
+
     // register routes
-    
+
     app.get("/users", async function(req: Request, res: Response) {
         return userRepository.find();
     });
-    
+
     app.get("/users/:id", async function(req: Request, res: Response) {
         return userRepository.findOne(req.params.id);
     });
-    
+
     app.post("/users", async function(req: Request, res: Response) {
         const user = userRepository.create(req.body);
         return userRepository.save(user);
     });
-    
+
+    app.put("/users/:id", function(req: Request, res: Response) {
+        return userRepository.update(req.params.id, req.body);
```

Comment:
I would like to avoid using `update` method. `update` method does a very simple thing - it executes update query and basically thats it. `save` method does everything orm is able to do. And many users, especially newcomers may not understand why update isnt working as they expected.

So in this case, if you want to have a separate `put` method I would recommend to `find` object by `req.params.id`, assign req.body and save that object.

Reply:
Sure,I understand your concern